src/ui-kit/experimental/date-range-v2/date-range-v2.component.html

```diff
@@ -0,0 +1,39 @@
+<sam-label-wrapper
```

Comment:
@yerramshilpa Since this is technically two fields, it needs to use the FieldsetWrapper and not the label wrapper. The label wrapper is for a single control, fieldset for multiple.

Reply:
Updated as required